src/main/java/edu/ucsb/cs156/frontiers/controllers/CoursesController.java

```diff
@@ -150,9 +191,39 @@ public ResponseEntity<Void> addInstallation(@Parameter(name = "installationId")
     public Object handleInvalidInstallationType(Throwable e) {
         return Map.of(
                 "type", e.getClass().getSimpleName(),
-                "message", e.getMessage()
-        );
+                "message", e.getMessage());
     }
 
+    /**
+     * This method returns a list of courses that the current user is enrolled.
+     * 
+     * @return a list of courses in the DTO form along with the student status in the organization.
+     */
+    @Operation(summary = "List all courses for the current student, including their org status")
+    @PreAuthorize("hasRole('ROLE_USER')")
+    @GetMapping("/list")
+    public List<Map<String, Object>> listCoursesForCurrentUser() {
+        Authentication authentication = SecurityContextHolder.getContext().getAuthentication();
+        OAuth2User user = (OAuth2User) authentication.getPrincipal();
+        String email = user.getAttribute("email");
```

Comment:
We would typically just write:
```suggestion
        String email = currentUser().getUser().getEmail();
```